acquia/acquia_cms
# ACMS-1387: Added the missing permission for frontend role for next js…

**Motivation**
* Add the missing permission to the frontend_preview_headless role - `issue subrequests`
Fixes #NNN

**Proposed changes**
* Added the missing permission to the frontend_preview_headless role - `issue subrequests`

**Alternatives considered**
* None

**Testing steps**
* Install the acquia_cms_headless kit and head towards permissions for - /admin/people/permissions/frontend_preview_headless and notice that missing permission is now added.
<img width="1522" alt="Screenshot 2022-09-06 at 3 03 41 PM" src="https://user-images.githubusercontent.com/15887127/188615593-b2ded83c-c005-4f7e-8edc-945bb31a6040.png">


**Merge requirements**
